etl/dbconn/connection_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+// +build integration
+
+package dbconn
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/block-explorer/configuration"
+	"github.com/insolar/block-explorer/testutils"
+)
+
```

Comment:
if tests require docker let's add a build constraint,  I suggest to add integration

Reply:
Yes, `+build integration`  set at line 6 above